schemas/sampler_v2_schema.json

```diff
@@ -0,0 +1,85 @@
+            "items": {
+                "type": "array",
+                "minItems": 1,
+                "prefixItems": [
+                    {"description": "The quantum circuit in QASM string or base64-encoded QPY format. See https://docs.quantum.ibm.com/api/qiskit/qpy for more details on QPY.", "type": "object"},
+                    {"description": "A dictionary of the parameter values. The keys are the names of the parameters, and the values are the actual parameter values.", "type": "object"},
+                    {"description": "The number of shots to use in this pub", "type": "integer"}
+                ]
+            }
+        },
+        "options": {
+            "type": "object",
+            "description": "Options for V2 Sampler",
+            "properties": {
+                "default_shots": {
+                    "description": "The default number of shots to use if none are specified in the PUBs",
+                    "type": "integer"
+                },
+                "dynamical_decoupling": {
+                    "description": "Suboptions for dynamical decoupling",
+                    "type": "object",
+                    "properties": {
+                        "enable": {
+                            "description": "Whether to enable DD as specified by the other options in this class",
+                            "type": "boolean"
+                        },
+                        "sequence_type": {
+                            "description": "Which dynamical decoupling sequence to use",
+                            "type": "string",
+                            "enum": ["XX", "XpXm", "XY4"]
+                        },
+                        "extra_slack_distribution": {
+                            "description": "Where to put extra timing delays due to rounding issues",
+                            "type": "string",
+                            "enum": ["middle", "edges"]
+                        },
+                        "scheduling_method": {
+                            "description": "Whether to schedule gates as soon as ('asap') or as late as ('alap') possible",
+                            "type": "string",
+                            "enum": ["alap", "asap"]
+                        }
+                    }
+                },
+                "execution": {
+                    "description": "Execution options",
+                    "type": "object",
+                    "properties": {
+                        "init_qubits": {
+                            "description": "Whether to reset the qubits to the ground state for each shot",
+                            "type": "boolean"
+                        },
+                        "rep_delay": {
+                            "description": "The delay between a measurement and the subsequent quantum circuit",
+                            "type": "number"
+                        }
```

Comment:
We have a new `meas_type` now 😁

Reply:
Done in `f6304e83ca29efb3c7af1cc9f6cd846e48856bb6`